src/Message/UpdatePaymentMethodRequest.php

```diff
@@ -20,6 +20,14 @@ public function getData()
             $data['options'] = $options;
         }
 
+        foreach (array('cardholderName', 'expirationDate') as $key) {

```

Comment:
if anyone has reservations about this sort of iteration, I'm open to alternatives (this is just the first thing that came to mind with a low LoC footprint